priv/cabbage/apps/itest/test/itest/transactions.exs

```diff
@@ -0,0 +1,174 @@
+defmodule TransactionsTests do
+  use Cabbage.Feature, async: true, file: "transactions.feature"
+
+  require Logger
+
+  alias Itest.Account
+
+  alias Itest.Client
+  alias Itest.Transactions.Currency
+
+  @num_accounts 50
+  setup do
+    {alices, bobs} =
+      @num_accounts
+      |> Account.take_accounts()
+      |> Enum.split(div(@num_accounts, 2))
+
+    %{alices: alices, bobs: bobs}
+  end
+
+  defwhen ~r/^they deposit "(?<amount>[^"]+)" ETH to the root chain$/,
+          %{amount: amount},
+          %{alices: alices} = state do
+    state =
+      alices
+      |> Enum.with_index()
+      |> Task.async_stream(
+        fn {{alice_account, _alice_pkey}, index} ->
+          initial_balance = Itest.Poller.eth_get_balance(alice_account)
+
+          key = String.to_atom("alice_initial_balance_#{index}")
+          data = [{key, initial_balance}]
+
+          {:ok, receipt_hash} =
+            amount
+            |> Currency.to_wei()
+            |> Client.deposit(alice_account, Itest.PlasmaFramework.vault(Currency.ether()))
+
+          gas_used = Client.get_gas_used(receipt_hash)
+          key = String.to_atom("alice_gas_used_#{index}")
+          data = [{key, gas_used} | data]
+
+          balance_after_deposit = Itest.Poller.eth_get_balance(alice_account)
+
+          key = String.to_atom("alice_ethereum_balance_#{index}")
+
+          data = [{key, balance_after_deposit} | data]
+          Map.put(state, String.to_atom("alice_data_#{index}"), data)
+        end,
+        timeout: 60_000,
+        on_timeout: :kill_task,
+        max_concurrency: @num_accounts
+      )
+      |> Enum.map(fn {:ok, result} -> result end)
+      |> Enum.reduce(%{}, fn state, acc -> Map.merge(state, acc) end)
+
+    {:ok, state}
+  end
+
+  defthen ~r/^they should have "(?<amount>[^"]+)" ETH on the child chain$/,
+          %{amount: amount},
+          %{alices: alices} = state do
+    alices
+    |> Enum.with_index()
+    |> Task.async_stream(
+      fn {{alice_account, alice_pkey}, index} ->
+        geth_block_every = 1
```

Comment:
that can be defined closer to the place where it's used for the first time,
please put it above `to_milliseconds`, or just name it `get_block_every_milliseconds` and put it instead of `to_milliseconds`